requirements.txt

```diff
@@ -2,6 +2,6 @@
 jsonschema
 networkx >= 3.0
 numpy >= 1.15.4
-scipy >= 1.1
+scipy >= 1.2
```

Comment:
If we don't need this anymore, I would revert to 1.1.

Reply:
🙏🏼